ACPs/236-continuous-staking/README.md

```diff
@@ -0,0 +1,154 @@
+exceeds the maximum stake limit defined in the network configuration, the excess amount is automatically withdrawn and
+sent to `ValidatorRewardsOwner` and `DelegatorRewardsOwner`.
+
+Because of the way `RewardValidatorTx` is structured, multiple instances cannot be issued without resulting in identical
+transaction IDs. To resolve this, a new transaction type has been introduced for both rewarding and stopping continuous
+validators: `RewardContinuousValidatorTx`. Along with the validator’s creation transaction ID, it also includes a
+timestamp. For simplicity and consistency, any stake exceeding the maximum limit is withdrawn from the validator, and
+the resulting UTXOs are tied to the `RewardContinuousValidatorTx` ID.
+
+Note: Submitting an `AddContinuousValidatorTx` immediately followed by a `StopContinuousValidatorTx` replicates
+the behavior of the current staking system.
+
+### New P-Chain Transaction Types
+
+The following new transaction types are introduced on the P-Chain to support this functionality:
+
+#### AddContinuousValidatorTx
+
+```golang
+type AddContinuousValidatorTx struct {
+  // Metadata, inputs and outputs
+  BaseTx `serialize:"true"`
+  
+  // Node ID of the validator
+  ValidatorNodeID ids.NodeID `serialize:"true" json:"nodeID"`
+  
+  // Period (in seconds).
+  Period uint64 `serialize:"true" json:"period"`
+  
+  // [Signer] is the BLS key for this validator.
+  Signer signer.Signer `serialize:"true" json:"signer"`
+  
+  // Where to send staked tokens when done validating
+  StakeOuts []*avax.TransferableOutput `serialize:"true" json:"stake"`
+  
+  // Where to send validation rewards when done validating
+  ValidatorRewardsOwner fx.Owner `serialize:"true" json:"validationRewardsOwner"`
+  
+  // Where to send delegation rewards when done validating
+  DelegatorRewardsOwner fx.Owner `serialize:"true" json:"delegationRewardsOwner"`
+  
+  // Fee this validator charges delegators as a percentage, times 10,000
+  // For example, if this validator has DelegationShares=300,000 then they
+  // take 30% of rewards from delegators
+  DelegationShares uint32 `serialize:"true" json:"shares"`
+  
+  // Weight of this validator used when sampling
+  Wght uint64 `serialize:"true" json:"weight"`
+}
+
+```
+
+#### StopContinuousValidatorTx
+
+```golang
+type StopContinuousValidatorTx struct {
+  // Metadata, inputs and outputs
+  BaseTx `serialize:"true"`
+  
+  // ID of the tx that created the continuous validator.
+  TxID ids.ID `serialize:"true" json:"txID"`
+  
+  // Authorizes this validator to be stopped.
+  // It is a BLS Proof of Possession signature of the TxID using validator key.
+  StopSignature [bls.SignatureLen]byte `serialize:"true" json:"stopSignature"`
+}
+```
+
+`StopSignature` is the BLS Proof of Possession signature of the tx ID of `AddContinuousValidatorTx` using the validator
+key.
+
+#### RewardContinuousValidatorTx
+
+```golang
+type RewardContinuousValidatorTx struct {
```

Comment:
At what point is this transaction initiated? when in the continuous staking flow does this transaction happen? 

It would be helpful to define a "flow of a continuous validator" to better understand the order of operations here